### PR TITLE
Fix session bug caused by sentry-go@v0.18.0

### LINKED ISSUE #348: Sessions broken in v2.1.9
Updating to the latest version of sentry broke sessions in the Playground.  It is because the middleware is called at two different places which affects the request context, causing the sessions not to work.

### PR DESCRIPTION
**Pull request by jribbink**
closes #348 

## Description

v2.1.9 of the playground API is not working in staging.  I have debugged this and determined that the sentry update that was mandatory as a part of updating to the latest version of Cadence was responsible.  It broke the session middleware

Namely, it has to do with how sentry affects the request context when being run twice in the middeware stack.  Sentry stores the current transaction in the context & if one exists already, then it uses this transaction.  However, this causes the request context to get rolled back to where it was during the first middleware call -> effectively removes any middleware values set between sentry middleware calls.

**TL;DR** this PR removes a redundant sentry middleware that was breaking the Playground.  I also removed gorilla/mux because we aren't using it and it was just being used for a type.
______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md).
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -170,7 +171,6 @@ func main() {
 
 		r.Use(httpcontext.Middleware())
 		r.Use(sessions.Middleware(cookieStore))
-		r.Use(monitoring.Middleware())
```
Comment (jribbink) [inline server/server.go:173]: the middleware is already called here https://github.com/onflow/flow-playground-api/pull/347/files#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4R113

**Review by devbugging — APPROVED** [review]

**Review by chasefleming — APPROVED** [review]
